compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -208,7 +209,7 @@ cmdInspect =
     detailOpt =
         fmap (maybe DA.Pretty.prettyNormal DA.Pretty.PrettyLevel) $
             optional $ option auto $ long "detail" <> metavar "LEVEL" <> help "Detail level of the pretty printed output (default: 0)"
-    cmd = execInspect <$> inputFileOpt <*> outputFileOpt <*> jsonOpt <*> detailOpt
+    cmd = execInspect <$> inputDalfOpt <*> outputFileOpt <*> jsonOpt <*> detailOpt
```

Comment:
`inputDalfOpt` isn’t accurate. `inspect` can also take a `.dar` file and will the inspect the main dalf.

Reply:
hmm, before it was asking for a DAML file, which is completely off...